include/wil/win32_helpers.h

```diff
@@ -20,7 +20,7 @@
 
 // detect std::bit_cast
 #ifdef __has_include
-#  if __has_include(<bit>)
+#  if (__cplusplus > 202000) && __has_include(<bit>)

```

Comment:
👍

Reply:
`/Zc:__cplusplus` isn't the default in MSVC or Visual Studio yet so I think this change should also check for `_MSVC_LANG` as well.

```suggestion
#  if (_MSVC_LANG >= 202002L || __cplusplus >= 202002L) && __has_include(<bit>)

```